metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,540 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test measures the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics/results will be got from the interconnection between
+# a client and a server using iperf tool.
+# The following cases are covered:
+#
+# case 1:
+#  container-server <----> container-client
+#
+# case 2"
+#  container-server <----> host-client
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/lib/network-common.bash"
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# Port number where the server will run
+port="${port:-5201}"
+fwd_port="${fwd_port:-$port:$port}"
+# Image name
+image="${image:-gabyct/network}"
+# Measurement time (seconds)
+transmit_timeout="${transmit_timeout:-5}"
```

Comment:
My suspicion is that we'll need a larger runtime/timeout for this in order to get stable repeatable results. I'll let you experiment and crunch the numbers to determine its effect.